### PR TITLE
#89 get questions from the mock server

### PR DESCRIPTION
**Pull request by visiteur85**
1. i got date from server to our component questions
2. i created unit test for question-reducer

https://github.com/gitkoshelew/CC.ui/issues/89

### CODE REVIEW

File: src/pages/testPage/index.tsx [hunk 1]
```diff
@@ -8,18 +8,34 @@ import { StylizedPaper } from '../../components/common/StylizedPaper/StylizedPap
...
   const nextQuestionHandler = useCallback(() => {
-    setIsRunning(true);
-    setCurrentTime(timeDefault);
-    alert(`${currentTime.minutes}:${currentTime.seconds}`);
-  }, [currentTime]);
+    // setIsRunning(true);
+    // setCurrentTime(timeDefault);
+    // alert(`${currentTime.minutes}:${currentTime.seconds}`);
+
+    if (numberOfQuestion === questions.length - 1) {
```
Comment (gitkoshelew) [inline src/pages/testPage/index.tsx:30]: you don't need if else.
if (numberOfQuestion === questions.length - 1) {
      alert('The end');
      setDisabled(true);
      return
 }
setQuestion(numberOfQuestion + 1);

File: src/pages/testPage/index.tsx [hunk 2]
```diff
@@ -45,23 +61,37 @@ const TestPage = () => {
...
         </StylizedPaper>
       </Stack>
     </Layout>
   );
-};
+}
+
+export const getServerSideProps = wrapper.getServerSideProps(
+  (store) => async () => {
+    const res = await questionsApi.getQuestions();
```
Comment (gitkoshelew) [inline src/pages/testPage/index.tsx]: api endpoint also should be called by redux

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/pages/testPage/index.tsx (hunk 1 above)
Comment (visiteur85) [reply src/pages/testPage/index.tsx:30]: I corrected it

File: src/pages/testPage/index.tsx (hunk 2 above)
Comment (visiteur85) [reply src/pages/testPage/index.tsx]: i understood you. I will use createAsyncThunk

File: src/Mocs/QuizMock.ts [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
-    title: 'document.getElement("p").innerHTML = "Hello World!"',
-    isDone: false,
-  },
-  {
-    id: 3,
-    title: 'document.getElementByName("p").innerHTML = "Hello World!"',
-    isDone: false,
-  },
-  { id: 4, title: 'demo.innerHTML = "Hello World!"', isDone: true },
+  // {
```
Comment (macroserP) [inline src/Mocs/QuizMock.ts]: Why did you keep it and not removed?

Comment (visiteur85) [reply src/Mocs/QuizMock.ts]: i've already moved it

**Review by gitkoshelew — APPROVED** [review]

**Review by macroserP — APPROVED** [review]

**Pull request merged**